contracts/deploy/Create3Address.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+import { CreateDeploy } from './CreateDeploy.sol';
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Create3Address contract
+ * @notice This contract can be used to deploy a contract with a deterministic address that depends only on
+ * the deployer address and deployment salt, not the contract bytecode and constructor parameters.
+ */
+contract Create3Address {
+    // keccak256(type(CreateDeploy).creationCode)
+    bytes32 internal constant DEPLOYER_BYTECODE_HASH = 0xf9bf726c56f6eb7a6a041cc888f1adb1231fafec3ca5392bf47fa10cf8df67fd;
```

Comment:
add a unit test for this

Reply:
It's covered by unit tests already, every half of the unit tests fail if it's wrong 😄